meta-oe/recipes-devtools/sqlite-orm/sqlite-orm_1.5.bb

```diff
@@ -0,0 +1,26 @@
+SUMMARY = "SQLite ORM light header only library for modern C++"
+HOMEPAGE = "https://github.com/fnc12/sqlite_orm"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=3b5bf5ee8bb16d8e19359efe11bdc3f2"
+
+inherit cmake
+
+DEPENDS += "sqlite3"
+
+PR = "r1"
+
+SRC_URI = "https://github.com/fnc12/sqlite_orm/archive/v1.5.tar.gz"
+SRC_URI[md5sum] = "f50ea3ff480f402bb7d96011643477c6"
+SRC_URI[sha256sum] = "4c77fc203954bfa48d6e9e89b386050599478f8e6c994d37e9a5a823261c52e6"
+
+S = "${WORKDIR}/sqlite_orm-1.5"
+
+EXTRA_OECMAKE += "-DSqliteOrm_BuildTests=OFF"
+
+FILES_${PN}-dev += "${includedir}/* ${libdir}/*"
```

Comment:
this seems redundant

Reply:
Project has cmake with install option and tries to install include/sqlite-orm.h and a few cmake file. If we don't add these folder in FILES, bitbake produces an error so I decided to add those files into dev package.